_docs/concepts/what-is-istio/comparison.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Istio vs. Other Software
+overview: A comparison between Istio's and similar software.
+              
+order: 30
+
+layout: docs
+type: markdown
+---
+
+This page provides a comparison between Istio and similar software.
+
+## Istio vs. Envoy
```

Comment:
This is confusing, since Envoy is part of Istio. Need to figure out how to retitle, since explaining how they relate is all goodness.

Maybe something like "Istio and Envoy"

Reply:
Makes sense